eng/pipelines/tools-repo-versioning.yml

```diff
@@ -0,0 +1,23 @@
+# Tag the (azure-sdk-tools) repo to create some form of versioning.
+trigger: none
+
+pr: none
+
+pool:
+  vmImage: windows-2019
+
+jobs:
+  - job: TagToolsRepo
+    displayName: Version azure-sdk-tools with Git Tag
+
+    steps:
+    - pwsh: |
+        $PushURL = "https://$(azuresdk-github-pat)@github.com/Azure/azure-sdk-tools.git"
+        $tag = "$(Build.Repository.Name)_$(Build.BuildNumber)"
+        Write-Host "$(Build.Repository.Name)
```

Comment:
While testing can you also see what the following variable outputs `$(Build.Repository.Uri)`? My hope is that it will be the exact uri you need except for the PAT and then we can do something like:

``` "$(Build.Repository.Uri)".Replace("https://", "https://$(azuresdk-github-pat)@")```

So that we can more easily reuse this yml file if we decided to do it for another fork or something.

Reply:
Yes, this works